<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationPaneClass</class>
 <widget class="QWidget" name="LocationPaneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LocationPane</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="locationTree">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="LocationProperties" name="propertiesWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="propertiesVisible">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Show Properties</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addItem">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeItem">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionCreateLocation">
   <property name="text">
    <string>Create Location ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LocationProperties</class>
   <extends>QWidget</extends>
   <header>LocationProperties.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>propertiesVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>propertiesWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>locationTree</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>LocationPaneClass</receiver>
   <slot>contextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>locationTree</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>LocationPaneClass</receiver>
   <slot>updateProperties(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>insertItemClicked()</slot>
  <slot>removeItemClicked()</slot>
  <slot>contextMenuRequested(QPoint)</slot>
  <slot>updateProperties(QTreeWidgetItem*)</slot>
 </slots>
</ui>
